integration_test/support/schemas.exs

```diff
@@ -76,6 +76,7 @@ defmodule Ecto.Integration.Comment do
   schema "comments" do
     field :text, :string
     field :lock_version, :integer, default: 1
+    field :nullable_lock_version, :integer, default: nil
```

Comment:
Instead of introducing a new field, can we simply use the previous field but we make sure to set it to nil when we first create the record?